mapproxy/util/ext/serving.py

```diff
@@ -119,8 +123,24 @@ def run_wsgi(self):
 
         def write(data):
             assert headers_set, 'write() before start_response'
+            import pdb; pdb.set_trace()
             if not headers_sent:
-                status, response_headers = headers_sent[:] = headers_set
+                if self.request_version == "HTTP/1.0":
+                    headers_sent[:] = headers_set
+                    status = headers_set[0]
+                    # remove non-HTTP1.0 headers
+                    response_headers = (
+                        header_tuple
+                        for header_tuple in headers_set[1]
+                        if header_tuple[0].lower() in [
+                                "date", "pragma",
+                                "location", "server", "www-authenticate",
+                                "allow", "content-Encoding", "content-length",
```

Comment:
please make everything lowercase in here ("content-Encoding") or it will not match, as you are comparing with lowerCase values

Reply:
yes, thanks